src/main.rs

```diff
@@ -183,6 +184,36 @@ fn get_pants_process() -> Result<Process> {
     })
 }
 
+fn get_pants_from_sources_process(pants_repo_location: PathBuf) -> Result<Process> {
+    let exe = pants_repo_location.join("pants").into_os_string();
+
+    let mut args = vec!["--no-verify-config".into()];
+    args.extend(env::args().skip(1).map(OsString::from));
+
+    let version = std::fs::read_to_string(
+        pants_repo_location
+            .join("src")
+            .join("python")
+            .join("pants")
+            .join("VERSION"),
+    )?;
+
+    // When running pants from sources you are likely to be modifying those sources, so you won't
+    // want pantsd running.  You can override this by setting ENABLE_PANTSD=true.
+    let enable_pantsd = env::var_os("ENABLE_PANTSD").unwrap_or_else(|| "false".into());
```

Comment:
This is actually super weird. Why not just use `PANTS_PANTSD`?! This comment is copied from the script, but of course no longer visible to the end user. I'll add support from reading from `PANTS_PANTSD` which is the obvious thing to do for an end user (or use the flag, which will work as well).

Reply:
I guess this was maybe some - misguided imo - attempt at letting users avoid the double dribble of PANTS_PANTSD? That is a real way to configure Pants though that is well documented - etc. Not to mention this is advanced use anyhow - changing the default for a pants from source run - so if any user can handle the awkward double dribble, its thie user in this mode's audience.

Reply:
Fixed.

Reply:
I suspect this wasn't a conscious attempt to avoid the double dribble, but that the script author didn't think to use the regular option env var for this.